content/pt-br/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Feed Oficial de CVE
+weight: 25
+outputs:
+  - json
+  - html 
+layout: cve-feed
+---
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+Esta é uma lista mantida pela comunidade oficial de CVEs anunciada pelo Comitê de Resposta de Segurança do Kubernetes. Veja [Informações de Segurança e Divulgação do Kubernetes](/docs/reference/issues-security/security/) para mais detalhes.
```

Comment:
Maybe you can use
```Esta é uma lista mantida pela comunidade, de CVEs oficiais anunciadas...```

Reply:
+1
@yagonobre do you feel that perhaps a comma should exist before "mantida"?
E.g.
> Esta é uma lista, mantida pela comunidade, de CVEs oficiais...